app/src/main/java/org/gateshipone/odyssey/activities/OdysseyMainActivity.java

```diff
@@ -215,7 +225,15 @@ protected void onCreate(Bundle savedInstanceState) {
                     break;
                 case R.id.nav_my_music:
                 default:
-                    fragment = MyMusicFragment.newInstance(getDefaultTab());
+                    if (mBackToSettings) {
```

Comment:
This will only work if the user doesn't change the start view.

Reply:
I think it would be better if you set the navId variable to R.id.nav_settings. So just add a check if navId should be set by the getDefaultView method or not and then handle the additional switch case.